x-pack/plugins/spaces/server/saved_objects/spaces_saved_objects_client.ts

```diff
@@ -175,32 +175,19 @@ export class SpacesSavedObjectsClient implements SavedObjectsClientContract {
    * @returns {promise} - { saved_objects: [{ id, type, version, attributes }], total, per_page, page }
    */
   public async find<T = unknown, A = unknown>(options: SavedObjectsFindOptions) {
-    throwErrorIfNamespaceSpecified(options);
```

Comment:
Not sure why this was there, there isn't a `namespace` property on `SavedObjectsFindOptions`

Reply:
Dead code! thanks for removing it